packages/loader/location-redirection-utils/src/resolveWithLocationRedirection.ts

```diff
@@ -29,23 +30,25 @@ export function isLocationRedirectionError(error: any): error is ILocationRedire
  * @param api - Callback in which user can wrap the loader.resolve or loader.request call.
  * @param request - request to be resolved.
  * @param urlResolver - resolver used to resolve the url.
+ * @param baseLogger - logger to send events.
  * @returns - Response from the api call.
  */
 export async function resolveWithLocationRedirectionHandling<T>(
 	api: (request: IRequest) => Promise<T>,
 	request: IRequest,
 	urlResolver: IUrlResolver,
-	logger?: ITelemetryLoggerExt,
+	baseLogger?: ITelemetryBaseLogger,
 ): Promise<T> {
 	let req: IRequest = request;
+	const logger = ChildLogger.create(baseLogger, "LocationRedirection");
```

Comment:
Haven't dig much, but what happens if baseLogger is undefined? Can we have a problem?